index-import.cpp

```diff
@@ -68,12 +68,14 @@ struct Remapper {
         // The format() function takes no explicit arguments, instead it gets
         // the values from the match object.
         auto substitution = match.format(replacement);
-        return match.prefix().str() + substitution + match.suffix().str();
+        auto output_str =
+            match.prefix().str() + substitution + match.suffix().str();
+        return path::remove_leading_dotslash(StringRef(output_str)).str();
```

Comment:
seems like we could probably optimize the swapping between string types here

Reply:
I'm open to suggestions. I agree it's ugly.

Reply:
yea i'd have to look at how the types interact, but I think it's also probably a perf hit, maybe doesn't matter

Reply:
maybe we'll do later